packages/playground/src/views/nodes.vue

```diff
@@ -153,6 +153,28 @@
         </template>
       </TfFilter>
 
+      <TfFilter
+        query-route="min-cpu"
+        v-model="filters.minCRU"
+        :rules="[
+          validators.isNumeric('This field accepts numbers only.'),
+          validators.min('The total number of CPUs should be larger then zero.', 1),
+          validators.validateResourceMaxNumber('This value is out of range.'),
+        ]"
+      >
+        <template #input="{ props }">
+          <VTextField label="Min CPU " variant="outlined" v-model="filters.minCRU" v-bind="props">
+            <template #append-inner>
```

Comment:
```suggestion
        <template #input="{ props }">
          <VTextField label="Min CPU (vCores)" variant="outlined" v-model="filters.minCRU" v-bind="props">
            <template #append-inner>
```

should we unify label? https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/playground/src/components/select_solution_flavor.vue#L27C31-L27C45

Reply:
yes, you're right.